WindowsServerDocs/storage/storage-spaces/drive-symmetry-considerations.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Drive symmetry considerations for Storage Spaces Direct 
+ms.author: cosdar
+ms.manager: eldenc
+ms.technology: storage-spaces
+ms.topic: article
+author: cosmosdarwin
+ms.date: 01/17/2018
+Keywords: Storage Spaces Direct
+ms.localizationpriority: medium
+---
+
+# Drive symmetry considerations for Storage Spaces Direct 
+
+> Applies To: Windows Server 2016
+
+[Storage Spaces Direct](storage-spaces-direct-overview.md) works best when every server has exactly the same drives.
+
+In reality, we recognize this is not always practical: Storage Spaces Direct is designed to run for years and to scale as the needs of your organization grow. Today, you may buy spacious 3 TB hard drives; next year, it may become impossible to find ones that small. Therefore, some amount of mixing-and-matching is supported.
+
+This topic explains the constraints and provides examples of supported and unsupported configurations.
+
+## Constraints
+
+### Type
+
+All servers must have the same [types of drives](choosing-drives.md#drive-types).
+
+For example, if one server has NVMe, they must *all* have NVMe.
+
+### Number
+
+All servers must have the same number of drives of each type.
+
+For example, if one server has *six* HDD, they must all have *six* HDD.
+
+   > [!NOTE]
+   > It is okay for the number of drives to differ temporarily during failures or while adding or removing drives.
+
+### Model
+
+It is recommended to use drives of the same model and firmware version whenever possible. If you can't, carefully select drives which are as similar as possible and verify that they don't have any conflicting requirements, such as system, adapter, or driver incompatibility.
+
+It is not advisable to mix-and-match drives of the same type with sharply different performance or endurance characteristics (unless one is cache and the other is capacity) because Storage Spaces Direct distributes IO evenly and does not discriminate based on model.
+
+### Size
+
+It is recommended to use drives of the same sizes whenever possible.
```

Comment:
Another instance of passive voice - I recommend saying "We recommend..." or just "Use drives of the same sizes..."